docs/Developer-Guide_Build-Options.md

```diff
@@ -31,6 +31,9 @@
 	- set to "yes" will prompt you right before the compilation starts to make changes to the source code. Separate for u-boot and kernel. It will also create a patch out of this. If you want that this patch is included in the normal run, you need to copy it to appropriate directory
 	- set to "no" compilation will run uninterrupted
 - **BUILD_ALL** (yes&#124;no&#124;demo): cycle through all available board and kernel configurations and make images for all combinations
+- **LIB_TAG** (master&#124;development):
```

Comment:
Is this a Tg or a branch ?

Reply:
It's actually a branch ... but IMO not critical since this is an "internal" switch. @zador-blood-stained  shell we update this accordingly?

Reply:
+1 to somehow tagging images and being able to rebuild a replica using that same tag.

This makes a lot of sense to people like me, who use customize_image.sh to release a modified version (in my case, [NCP](nextcloudplus.com)). This would allow me to gather people from the NCP community and do some testing of WIP images and if that happens, we can all benefit.

For instance, I couldn't reproduce the stable rock64 image -> [link](https://forum.armbian.com/topic/6956-rock64-debian-building-a-replica-of-current-testing-image/)

I don't know if this is being discussed in the forums somewhere of what is the right place